### PR TITLE
Update Android build tools

### PR DESCRIPTION
**Pull request by janisozaur**

### CODE REVIEW

File: src/openrct2-android/app/build.gradle [hunk 1]
```diff
@@ -45,28 +45,25 @@ android {
...
-            ndk {
-                abiFilters 'armeabi-v7a'
-            }
-        }
         arm {
             ndk {
                 abiFilters 'armeabi-v7a', 'arm64-v8a'
             }
         }
-        x86 {
```
Comment (AaronVanGeffen) [inline src/openrct2-android/app/build.gradle]: I take it you wish to focus on building for ARM only?

Given the Intel market share for Android tablets, or lack thereof, I don't see a problem with the change; I'd just like to make sure.

Comment (janisozaur) [reply src/openrct2-android/app/build.gradle]: oh, that was not meant to be part of PR

Comment (janisozaur) [reply src/openrct2-android/app/build.gradle]: ChromeOS is largely x86/x86-64 and they can run Android applications

Comment (AaronVanGeffen) [reply src/openrct2-android/app/build.gradle]: Ah, right! Well, glad we're doing code review, then! :+1:

**Comment by Gymnasiast** [comment]
@janisozaur Could you update this PR, so it can be merged?

**Pull request merged**